### PR TITLE
ConsoleInteraction.py: Add param docs for methods

### LINKED ISSUE #3452: ConsoleInteraction.py: Add param docs for functions

### PR DESCRIPTION
**Pull request by vijeth-aradhya**
Add param docs to all the functions, with appropiate
function description and parameter description.

Closes #3452

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### Reviewers

@adtac

### CODE REVIEW

**Comment by vijeth-aradhya** [comment]
@adtac There are two trivial functions - `highlight_text` and `format_lines` I haven't added any param doc for them since they are small and readable. If you suggest we should, then I will do it :+1: 
And, also `*args` in `print_results_formatted` method is _not_ used at all :| What should I do for that? It just contains the arguments, the input file as a string, and two other things

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -289,6 +295,14 @@ def print_results_formatted(log_printer,
                             section,
                             result_list,
                             *args):
+    """
+    Prints results in the user given format
```
Comment (Mixih) [inline coalib/output/ConsoleInteraction.py]: elaborate that it accepts a format string from the format setting, etc

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -272,6 +272,12 @@ def print_result(console_printer,
 
 
 def print_diffs_info(diffs, printer):
+    """
+    Prints diffs information (additions and deletions) to the console
```
Comment (Mixih) [inline coalib/output/ConsoleInteraction.py]: "additions and deletions" -> "number of additions and deletions"

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (NiklasMM) [inline coalib/output/ConsoleInteraction.py]: Theres a `.` missing

Comment (Mixih) [inline coalib/output/ConsoleInteraction.py]: I think *args is there to sponge up any extra param that may get specified by a unified caler, so it needs no docs

File: coalib/output/ConsoleInteraction.py [hunk 3]
```diff
@@ -289,6 +295,14 @@ def print_results_formatted(log_printer,
                             section,
                             result_list,
                             *args):
+    """
+    Prints results in the user given format
+
+    :param log_printer:    Printer responsible for logging the messages.
+    :param section:        The section to which the results belong to.
```
Comment (NiklasMM) [inline coalib/output/ConsoleInteraction.py]: I think the last `to` can be removed

File: coalib/output/ConsoleInteraction.py [hunk 4]
```diff
@@ -272,6 +272,12 @@ def print_result(console_printer,
 
 
 def print_diffs_info(diffs, printer):
+    """
+    Prints diffs information (additions and deletions) to the console
+
+    :param printer:  Object responsible for printing diffs on console.
+    :param diffs:    List of Diff objects containing corresponding diff info.
```
Comment (NiklasMM) [inline coalib/output/ConsoleInteraction.py]: Please swap these lines so the order corresponds to that of the arguments.

**Comment by vijeth-aradhya** [comment]
@Mixih @NiklasMM Is it fine now :)

**Comment by aptrishu** [comment]
ack 6d660b

**Comment by aptrishu** [comment]
@rultor merge

**Pull request merged**